dolphin/drivers/huawei/oceanstor/rest_client.py

```diff
@@ -0,0 +1,204 @@
+# Copyright 2020 The SODA Authors.
+# Copyright (c) 2016 Huawei Technologies Co., Ltd.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import json
+
+from oslo_log import log as logging
+import requests
+import six
+
+from dolphin import exception
+from dolphin.i18n import _
+from dolphin.drivers.huawei.oceanstor import consts
+
+LOG = logging.getLogger(__name__)
+
+
+class RestClient(object):
+    """Common class for Huawei OceanStor storage system."""
+
+    def __init__(self, **kwargs):
+        # self.configuration = configuration
+        host = kwargs.get('host', 'localhost')
+        port = kwargs.get('port', '8088')
+        self.san_address = [
```

Comment:
Hi @joseph-v  Could you plz explain why need a list here?

Reply:
The rest_client.py has an option to try multiple addresses for authorization, in case of failure. Currently we use only one url.

Reply:
Done